app/controllers/organizations/staff/external_form_upload_controller.rb

```diff
@@ -12,6 +12,11 @@ def index
       def create
         authorize! :external_form_upload, context: {organization: Current.organization}
         file = params[:files]
+
```

Comment:
This can be added to the `module AttachmentManageable`. Given it's something that we would want to use across multiple controllers we should have it in a module, then call it in a `before_action` on the `#create` method. 

Something like 
```
def handle_wrong_file_format_for_csv
  unless params[:files].content_type == "text/csv
    redirect_to staff_external_form_upload_index_path, alert: "File must be a CSV."
   end
end
```

And call it on the `class ExternalFormUploadController` **after** the `before_action :allow_only_one_attachment, only: [:create]` on the `#create` action.